FirebaseAppCheck/Interop/FIRAppCheckInterop.h

```diff
@@ -43,6 +43,14 @@ NS_SWIFT_NAME(AppCheckInterop) @protocol FIRAppCheckInterop
 /// `tokenDidChangeNotificationName`.
 - (NSString *)notificationAppNameKey;
 
+// MARK: - Optional API
```

Comment:
Add TODO/issue to make required in 11.0?

Reply:
great idea.

Reply:
Opened #11284 to track.